heron/instance/src/java/com/twitter/heron/instance/HeronInstance.java

```diff
@@ -257,6 +260,12 @@ public static void main(String[] args) throws IOException {
     String overrideConfigFile
         = commandLine.getOptionValue(CommandLineOptions.OVERRIDE_CONFIG_FILE);
 
+    Integer remoteDebuggerPort = null;
+    if (commandLine.hasOption(CommandLineOptions.REMOTE_DEBUGGER_PORT)) {
+      remoteDebuggerPort = Integer.parseInt(
```

Comment:
once heron-instance listens to this port, what requests will it accept?

Reply:
a user can attach a remote debugger to the port